crates/evm/execution-types/src/execution_outcome.rs

```diff
@@ -168,13 +168,13 @@ impl ExecutionOutcome {
     }
 
     /// Transform block number to the index of block.
-    fn block_number_to_index(&self, block_number: BlockNumber) -> Option<usize> {
+    pub fn block_number_to_index(&self, block_number: BlockNumber) -> Option<usize> {
         if self.first_block > block_number {
-            return None
+            return None;
```

Comment:
please revert all unrelated formatting changes. the semicolon is added when you run `cargo fmt` instead of `cargo +nightly fmt`!
```suggestion
            return None
```